Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:current-alpine
+
+WORKDIR /app
+COPY package*.json ./
+
+RUN apk add libxml2 libxslt git imagemagick python3 make g++
+RUN npm install
+RUN npm install -g grunt-cli grunt modern-syslog
```

Comment:
All of these are dependencies; it should be enough to install `grunt-cli` globally so that `grunt prepare` works.

Or maybe even try skipping this whole line and instead change `RUN grunt prepare` below to `RUN npx grunt prepare`?